### DISCUSSION TITLE
Add list of resumes for inspirations in our GitHub Organisation

### DISCUSSION THREAD

**Discussion by vinzvinci** [EddieHubCommunity/support#627] [Ideas]
![newideas](https://user-images.githubusercontent.com/73097560/103482535-5b22f300-4e1c-11eb-92bd-12ab54ad1bb1.png)

Let me know your thoughts

**Comment by Vyvy-vi** [comment]
That would be great :+1:

**Comment by eddiejaoude** [comment]
Oh this would be great. Are there any existing examples online of people who have tried this already?

**Reply by vinzvinci** [reply]
I didn't see one yet when I'm browsing.

**Reply by Akshu-on-github** [reply]
I've seen a few in passing, lemme see if I can find them now...

**Comment by vinzvinci** [comment]
I saw one, but we could make it better.

Link: <a href="https://github.com/justinabrahms/junior-dev-resumes">https://github.com/justinabrahms/junior-dev-resumes</a>

<hr/> 

Should we add **GitHub Markdown and GitHub pages** for this idea?

<hr/>

**We can separate resume:**
- Fresher
- Experience
